cmd/meek-server/smoketest/socks5.sh

```
@@ -0,0 +1,140 @@
+    args+=(--data-binary "")
+  fi
+  curl "${args[@]}" "$POST_URL"
+}
+
+# Send <payload-file> then poll empty POSTs until at least <min-bytes> received,
+# concatenating each response chunk into <accum-file>.
+# usage: send_and_drain <payload-file> <accum-file> <min-bytes> <max-polls>
+send_and_drain() {
+  local payload=$1 accum=$2 minb=$3 maxp=$4
+  : > "$accum"
+  local tmp=$(mktemp)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)